response_operations_ui/controllers/sign_in_controller.py

```diff
@@ -0,0 +1,17 @@
+import json
+import logging
+
+import requests
+from structlog import wrap_logger
+
+from response_operations_ui import app
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+
+def get_sign_in(sign_in_data):
+    logger.debug('Retrieving sign in authorisation')
+    url = f'{app.config["BACKSTAGE_API_URL"]}/sign-in-uaa'
```

Comment:
why is uaa in the url? shouldn't it just be sign-in?

Reply:
We added this as a new resource, but I think we can remove the old sign-in, and if we can, just replace with this

Reply:
We cannot remove the old sign-in endpoint. It's being used by the angular application which is getting used for BRES